sys/include/net/skald/ibeacon.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_skald_ibeacon Skald about iBeacon
+ * @ingroup     net_skald
+ * @brief       Skald's simple iBeacon abstraction
+ *
+ * # About
+ * This Skald module supports the creation and advertisement of BLE iBeacons as
+ * defined by Apple (see https://developer.apple.com/ibeacon/).
```

Comment:
Same proposal here here:

```markdown
This Skald module supports the creation and advertisement of BLE [iBeacons](https://developer.apple.com/ibeacon/) as defined by Apple.
```

Reply:
NACK

Reply:
the NACK was superfluous, github interface didn't show my first comment....